webpack.base.config.js

```diff
@@ -1,76 +1,54 @@
-var path = require('path');
-var nodeModulesDir = path.resolve(__dirname, 'node_modules');
-var BundleTracker = require('webpack-bundle-tracker');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+const path = require('path');
 
-module.exports = [
-  {
-    entry: ['./frontend/js/jquery-index.js'],
-    output: {
-      path: path.resolve('./frontend/bundles/'),
-      filename: 'bundle-jquery.js',
-    },
-    module: {
-      rules: [
-        {
-          test: /\.jsx?$/,
-          exclude: [nodeModulesDir],
-          use: {
-            loader: 'babel-loader',
-            options: {
-              presets: ['@babel/preset-env'],
-            },
+const nodeModulesDir = path.resolve(__dirname, 'node_modules');
+
+const devMode = process.env.NODE_ENV !== 'production';
+
+module.exports = {
```

Comment:
This should be the same as the previous one, I think there should be no changes for the purpose of this pr